### PR TITLE
Add the Hasslehof

### PR DESCRIPTION
**Pull request by couto**

### CODE REVIEW

**Comment by tallnato** [comment]
ahahah 😄👏

**Review by tallnato — DISMISSED** [review]
ahahah 😄👏

LGTM

File: README.md [hunk 1]
```diff
@@ -1,2 +1,40 @@
...
+```bash
+crontab -e
+```
+
+```bash
+0  11  *  *  * bash ~/.list_videos.sh
+```
+
+```bash
+#!/usr/bin/env bash
```
Comment (portellaa) [inline README.md]: why not just `#!/bin/bash` 😂 💄 

just kidding, i was just messing with you 😁

Comment (portellaa) [inline README.md]: why not just `#! /bin/bash` ❓❗️ 😂 😁 

just messing with you 😉

File: README.md [hunk 2]
```diff
@@ -1,2 +1,40 @@
 # Prank-amos
 List of pranks that you can do to your office mates 😁
+
```
Comment (tallnato) [inline README.md:3]: Soo.... we need to define a structure for the list of pranks.

Talking with @portellaa we were thinking that the main `README.md` could have the list of pranks and then each prank would have its folder with the `README.md` there.

What do you think?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,40 @@
...
+* [David Hasselhoff - True Survivor (from Kung Fury)][1]
+* [David Hasselhoff - Hooked on a feeling][2]
+* [David Hasselhodd - Jump in my car][3]
+
+[1]: https://www.youtube.com/watch?v=ZTidn2dBYbY
+[2]: https://www.youtube.com/watch?v=PJQVlVHsFF8
+[3]: https://www.youtube.com/watch?v=dm7jEA3frY4
+
+```bash
+crontab -e
```
Comment (portellaa) [inline README.md]: `EDITOR=vim` or the favourite editor? 🤷‍♂️ but is fine by me 😁

File: README.md [hunk 4]
```diff
@@ -1,2 +1,40 @@
...
+
+# this is the ~/.list_videos.sh file
+
+VIDEOS=("https://www.youtube.com/watch?v=ZTidn2dBYbY" "https://www.youtube.com/watch?v=PJQVlVHsFF8" "https://www.youtube.com/watch?v=dm7jEA3frY4")
+
+size=${#VIDEOS[@]}
+index=$(($RANDOM % $size))
+
+open "${VIDEOS[$index]}"
+```
```
Comment (tallnato) [inline README.md]: Should we add how to cancel this? 

No, right? 😁😁😁

**Review by portellaa — APPROVED** [review]
Our first contribution 🎉 😁

File: README.md (hunk 4 above)
Comment (portellaa) [reply README.md]: of course not 😂

File: README.md (hunk 2 above)
Comment (couto) [reply README.md:3]: Moved the prank to it's own file and added the link in this page

Comment (portellaa) [reply README.md:3]: Let's discuss this 🤔 

perhaps with links, placing this files inside folders

File: README.md (hunk 1 above)
Comment (couto) [reply README.md]: compatibility issues between different *BSD and Linux

File: README.md (hunk 3 above)
Comment (couto) [reply README.md]: There's a button to automatically suggest new code. Use it!

**Review by tallnato — APPROVED** [review]
LGTM 🤘🤘🤘🤘

**Comment by portellaa** [comment]
Go ahead and merge this 💪

**Pull request merged**